interpreter/interface.go

```diff
@@ -633,3 +634,7 @@ func (ctx NoOpStringContext) IsTypeInfoRecovered(_ common.Location) bool {
 func (ctx NoOpStringContext) SemaTypeFromStaticType(_ StaticType) sema.Type {
 	panic(errors.NewUnreachableError())
 }
+
+func (ctx NoOpStringContext) SemaAccessFromStaticAuthorization(auth Authorization) sema.Access {
```

Comment:
Maybe also remove the other unused receiver variables from the other methods

```suggestion
func (NoOpStringContext) SemaAccessFromStaticAuthorization(auth Authorization) sema.Access {
```